{
    "topic": "TS LF Trading Bot Errors",
    "pageNumber": "50",
    "type": "TS LF Trading Bot Error - Snapshot Unexpected Error",
    "definition": {
        "text": "The Trading Bot found an Unexpected Error during the generation of a Snapshot. The trading process continued but the task of generating the Snapshot was aborted.",
        "updated": 1611681073241
    },
    "paragraphs": [
        {
            "style": "Error",
            "text": "Snapshot Unexpected Error",
            "updated": 1611680863486
        },
        {
            "style": "Title",
            "text": "Error Details"
        },
        {
            "style": "Text",
            "text": "In the following section you will find the specific information received when this error occured."
        },
        {
            "style": "Text",
            "text": "You will only see it if you opened this page as a result of the error actually happening.",
            "updated": 1611680880668
        },
        {
            "style": "Subtitle",
            "text": "Error Message"
        },
        {
            "style": "Text",
            "text": "The following is the Error Message of the Exception that occured at the LF Trading Bot:"
        },
        {
            "style": "Placeholder",
            "text": "errorMessage"
        },
        {
            "style": "Subtitle",
            "text": "Error Stack"
        },
        {
            "style": "Text",
            "text": "The following is the Error Stack of the Exception that occured at the LF Trading Bot:"
        },
        {
            "style": "Placeholder",
            "text": "errorStack"
        },
        {
            "style": "Subtitle",
            "text": "Error Code"
        },
        {
            "style": "Text",
            "text": "The follwing is the Error Code of the Exception that occured at the LF Trading Bot:"
        },
        {
            "style": "Placeholder",
            "text": "errorCode"
        },
        {
            "style": "Subtitle",
            "text": "Error Details"
        },
        {
            "style": "Text",
            "text": "The follwing are all the Error details of the Exception that occured at the LF Trading Bot:"
        },
        {
            "style": "Placeholder",
            "text": "errorDetails"
        },
        {
            "style": "Subtitle",
            "text": "Node Name"
        },
        {
            "style": "Text",
            "text": "The follwing is the node's name of the Node most related to this error:"
        },
        {
            "style": "Placeholder",
            "text": "nodeName"
        },
        {
            "style": "Subtitle",
            "text": "Node Code"
        },
        {
            "style": "Text",
            "text": "The follwing is the code stored at the node that produced the error at the LF Trading Bot:"
        },
        {
            "style": "Placeholder",
            "text": "nodeCode"
        }
    ]
}